test/parallel/test-child-process-validate-stdio.js

```diff
@@ -30,6 +29,19 @@ common.expectsError(() => _validateStdio(stdio2, true),
                     { code: 'ERR_IPC_SYNC_FORK', type: Error }
 );
 
+// Should throw if stdio is not a valid input
+{
+  const stdio = ['foo'];
+  common.expectsError(() => _validateStdio(stdio, false),
+                      { code: 'ERR_INVALID_SYNC_FORK_INPUT', type: TypeError }
+  );
+}
+
+// Should throw if stdio is not a valid option
+{
+  const stdio = [{ foo: 'bar' }];
```

Comment:
I think it would be better if we can collect all invalid options which both throw `expectedError` here(for reduce duplicated code). 

Things like:

```js
['foo', 600, [{ foo: 'bar' }]].forEach((stdio) => {
  common.expectsError(() => _validateStdio(stdio), expectedError);
})
```

But it shouldn't block this :)